scripts/ai_pr_reviewer/test/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
```

Comment:
is the `/test` dir necessary? i think we can test by running directly on a PR. maintaining it always is a pita